README.md

```diff
@@ -102,6 +102,22 @@ Note, that you should pass a valid input method name to `fcitx-remote`
 command. The last layout in the example uses the
 [Mozc](https://github.com/google/mozc) as an input method for Japanese input.
 
+## How to keep chosen layout per window
+
+If you wish to remember selected layout per window set `remember_layout` parameter while creating instance of the plugin:
+
+```
+local kbdcfg = keyboard_layout.kbdcfg({remember_layout=true})
+```
+
+You can also set default layout for new windows with `default_layout_index` variable like that:
+
+```
+local kbdcfg = keyboard_layout.kbdcfg({remember_layout=true, default_layout_index=1})
+```
+
+Index depends on your `kbdcfg.add_primary_layout` calls.
```

Comment:
I suggest to create separate section for `default_layout_index` because we can use it to set `kbdcfg.current` and in this case it is just the index of default layout and it doesn't depends on `kbdcfg.add_primary_layout`.

And also, it is necessary to add something like that: "Index of layout is set in the order of adding layouts".

I'm not sure that it is a good idea to create a lot of sections for options, functions, usage and etc. I'll think how organize documentation in better way. @KryDos May be do you have any inputs?